### PR TITLE
Covariance param

### LINKED ISSUE #40: Orientation Covariance
Currently no covariance is provided for the output orientations. Short of a proper covariance propagation from the original sensor sources, I'd like to stub in a parametrized reasonable fixed covariance matrix, so that the output can be used with robot_localization and robot_pose_ekf

### PR DESCRIPTION
**Pull request by paulbovbel**
Fix #40

### CODE REVIEW

File: imu_filter_madgwick/src/imu_filter.cpp [hunk 1]
```diff
@@ -51,6 +51,10 @@ ImuFilter::ImuFilter(ros::NodeHandle nh, ros::NodeHandle nh_private):
   if (!nh_private_.getParam ("publish_debug_topics", publish_debug_topics_))
     publish_debug_topics_= false;
 
+  double orientation_stddev;
+  nh_private_.param<double>("orientation_stddev", orientation_stddev, 0.0);
+  orientation_variance_ = orientation_stddev * orientation_stddev;
+
```
Comment (mintar) [inline imu_filter_madgwick/src/imu_filter.cpp]: You don't need lines 54-57. It's automatically taken care of by dynamic reconfigure. What happens is:
- we register the dynamic reconfigure server (L75)
- it reads the parameters and calls `reconfigCallback()`

This happens before we initialize the subscribers, so we can be sure that the param has been processed before we first access `orientation_variance_`.

**Comment by mintar** [comment]
Other than my comment, this looks good. Let me know when I can merge it.

File: imu_filter_madgwick/src/imu_filter.cpp (hunk 1 above)
Comment (paulbovbel) [inline imu_filter_madgwick/src/imu_filter.cpp]: Thanks, I haven't used dynamic reconfigure much. Can you still set initial values in a launch file without 54-57?

Comment (mintar) [inline imu_filter_madgwick/src/imu_filter.cpp]: Yes, you can! Dynamic reconfigure reads the parameters from the parameter server on initialization.

**Comment by paulbovbel** [comment]
Beauty, ready for pull. That's the last from me, could you please cut a release, it would help me greatly in setting everything up on our Husky.

**Comment by mintar** [comment]
Done: https://github.com/ros/rosdistro/pull/7013

**Pull request merged**